{"name": "Проверка ошибки логина несуществующего пользователя", "status": "passed", "start": 1755162169651, "stop": 1755162169794, "uuid": "0a732a39-dcc8-4283-bbc8-74a3f1006d4e", "historyId": "43fec8d83301d39d003571dbbdefa737", "testCaseId": "43fec8d83301d39d003571dbbdefa737", "fullName": "tests.test_login_user.TestUserLogin#test_login_not_exist_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "DESKTOP-27LVRII"}, {"name": "thread", "value": "14204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}], "titlePath": ["tests", "test_login_user.py", "TestUserLogin"]}